lib/utils.js

```diff
@@ -451,6 +451,10 @@ function deepMergeProperty(base, extras) {
   return mergedProperty;
 }
 
+const numberIsFinite = Number.isFinite || function(value) {
+  return typeof value === 'number' && isFinite(value);
```

Comment:
@bajtos where is the `isFinite()` function from? You may need to import it from somewhere.